src/allmydata/util/cputhreadpool.py

```diff
@@ -46,14 +46,37 @@
 P = ParamSpec("P")
 R = TypeVar("R")
 
+# Is running in a thread pool disabled? Should only be true in synchronous unit
+# tests.
+_DISABLED = False
+
 
 def defer_to_thread(
-    reactor: IReactorFromThreads, f: Callable[P, R], *args: P.args, **kwargs: P.kwargs
+    f: Callable[P, R], *args: P.args, **kwargs: P.kwargs
```

Comment:
Isn't un-parameterizing the reactor a step backwards?

Reply:
The alternative was scattering "from twisted.internet import reactor" throughout the code in ways that I am starting to think are unnecessary.